code/TestAutomation.Tests.BDD/Steps/PageSteps/CareersPageSteps.cs

```diff
@@ -0,0 +1,44 @@
+﻿using TechTalk.SpecFlow;
+using TestAutomation.Epam.PageObjects.Pages;
+
+namespace TestAutomation.Tests.BDD.Steps.PageSteps
+{
+    [Binding]
+    public class CareersPageSteps : BaseStep
+    {
+        protected CareersPage CareersPage { get; set; }
+        public CareersPageSteps()
+        {
+            CareersPage = new CareersPage(Driver);
+        }
+
+        [Given(@"I open Careers page with click on '([^']*)' button")]
```

Comment:
Please review params in step. All method’s params should be used. If no -> [Given(@"I open Careers page with click on 'Careers' button")]

Reply:
done